### PR TITLE
Enable adding and removing reactions

### PR DESCRIPTION
**Pull request by sebastinez**
Add the reaction selector back and be able to react to comments

Missing:

- [x] Adding tests
- [x] [Patch to add ability to remove reactions on `radicle-httpd`](https://app.radicle.xyz/nodes/seed.radicle.xyz/rad:z3gqcJUoA1n9HaHKufZs5FCSGazv5/patches/6aab1d036354378edfaaaa9347b5f40d2c852302)
- [x] Get UI to an acceptable state
- [x] Remove a reaction by clicking on it in the reaction selector
- [x] Add tests for multiple peers reaction on the same comment..~
  - Did test it manually and worked, will open an issue to add multiple peers doing changes simultaneously.

Successor of #998

### CODE REVIEW

**Comment by rudolfs** [comment]
Screenshot 2023-08-23 at 09 11 36

Can we make all reactions look the same? I prefer the look of the top (issue) reactions which are slightly smaller. The text is also smaller and blue.

File: src/components/Comment.svelte [hunk 1]
```diff
@@ -99,17 +124,39 @@
     {:else}
       <Markdown {rawPath} content={body} />
     {/if}
-    {#if groupedReactions.size > 0}
-      <div class="reactions">
-        {#each groupedReactions as [reaction, nids], key}
-          <Chip {key}>
-            <div class="reaction">
+    <div class="reactions">
+      {#if $httpdStore.state === "authenticated"}
```
Comment (rudolfs) [inline src/components/Comment.svelte:128]: Can we move this check one element above so it wraps the reaction container?

**Comment by rudolfs** [comment]
When not logged in, can we show the reactions just read-only? Like we do for the issue reactions:

Screenshot 2023-08-23 at 09 39 16

File: src/components/Comment.svelte (hunk 1 above)
Comment (sebastinez) [reply src/components/Comment.svelte:132]: I think we should keep it like this, so we can show the reactions to a comment, even if a user isn't authenticated.

**Comment by sebastinez** [comment]
> Screenshot 2023-08-23 at 09 11 36
> Can we make all reactions look the same? I prefer the look of the top (issue) reactions which are slightly smaller. The text is also smaller and blue. The reactions could also benefit from a `user-select: none` css attribute, so that clicking them doesn't select any text.

Yeah no problem, done

**Comment by sebastinez** [comment]
> When not logged in, can we show the reactions just read-only? Like we do for the issue reactions:
> 
> Screenshot 2023-08-23 at 09 39 16

Yeah works for me

**Comment by rudolfs** [comment]
There's a weird margin now showing up in the visual tests:

Screenshot 2023-08-23 at 14 57 52

**Comment by sebastinez** [comment]
Visual changes, still show the issue icon change

**Comment by sebastinez** [comment]
> There's a weird margin now showing up in the visual tests:
> 
> Screenshot 2023-08-23 at 14 57 52

Is fixed with [3fdd989](https://github.com/radicle-dev/radicle-interface/pull/1008/commits/3fdd98994d1ae99dc3ac413aa19c686084668568)

**Comment by rudolfs** [comment]
There seems to be a margin missing between the reactions: 
Screenshot 2023-08-23 at 15 39 30

**Review by rudolfs — APPROVED** [review]

**Pull request merged**